controller/internal/supervisor/iptablesctrl/acls.go

```diff
@@ -247,6 +247,15 @@ func (i *iptables) generateACLRules(cfg *ACLInfo, rule *aclIPset, chain string,
 			"-m", "set", "--match-set", rule.ipset, ipMatchDirection,
 		}
 
+		if proto == constants.AllProtoString {
+			iptRule = []string{
+				appPacketIPTableContext,
+				chain,
+				"!", "-p", "tcp",
+				"-m", "set", "--match-set", rule.ipset, ipMatchDirection,
+			}
+		}
+
```

Comment:
What happens with UDP? I think we will break it as well. I don't see how this can be done without moving the global ACLs around.

Reply:
Moving the global acls causes other issues, where pu-pu traffic hits the ext net rule.

Reply:
I moved the udp nfq rules on top of the pu chain.